ocaml/xapi/xapi_sr.ml

```diff
@@ -424,22 +424,42 @@ let maybe_copy_sr_rrds ~__context ~sr =
     with Rrd_interface.Rrdd_error(Archive_failed(msg)) ->
       warn "Archiving of SR RRDs to stats VDI failed: %s" msg
 
+let forget_common ~__context ~sr =
```

Comment:
It looks like "delete", "forget", and "destroy" are quite synonymous here. The metrics are destroyed/removed but the underlying devices of course not. The "common" part in the name is not obvious - metrics with matching names are removed but this reflects internal workings - I'd say "remove_inactive_dss"

Reply:
The device has been removed at this point. Here we are updating the xapi database and metrics to reflect this. I'll try to think of a better name.